kernel/src/fs/utils/inode.rs

```diff
@@ -80,6 +80,43 @@ impl From<DeviceType> for InodeType {
     }
 }
 
+bitflags! {
+    pub struct RequestPermission: u16 {
+        // This implementation refers the implementation of linux
+        // https://elixir.bootlin.com/linux/v6.0.9/source/include/linux/fs.h#L95
+        const MAY_EXEC		= 0x0001;
+        const MAY_WRITE		= 0x0002;
+        const MAY_READ		= 0x0004;
+        const MAY_APPEND    = 0x0008;
+        const MAY_ACCESS	= 0x0010;
+        const MAY_OPEN		= 0x0020;
+        const MAY_CHDIR		= 0x0040;
+        const MAY_NOT_BLOCK	= 0x0080;
+    }
+}
+impl RequestPermission {
+    pub fn request_read(&self) -> bool {
```

Comment:
Is `allow_read()` or `can_read()` a better function name?

Reply:
Done